tests/files/pcsd.crt

```diff
@@ -0,0 +1,26 @@
+-----BEGIN CERTIFICATE-----
+MIIEXjCCAsagAwIBAgIGAXWdrKfzMA0GCSqGSIb3DQEBCwUAMG4xCzAJBgNVBAYT
+AlVTMQswCQYDVQQIDAJNTjEUMBIGA1UEBwwLTWlubmVhcG9saXMxDTALBgNVBAoM
+BHBjc2QxDTALBgNVBAsMBHBjc2QxHjAcBgNVBAMMFWxvY2FsaG9zdC5sb2NhbGRv
+bWFpbjAiGA8yMDIwMTEwNjEzMDg1OVoYDzIwMzAxMTA0MTMwODU5WjBuMQswCQYD
+VQQGEwJVUzELMAkGA1UECAwCTU4xFDASBgNVBAcMC01pbm5lYXBvbGlzMQ0wCwYD
+VQQKDARwY3NkMQ0wCwYDVQQLDARwY3NkMR4wHAYDVQQDDBVsb2NhbGhvc3QubG9j
+YWxkb21haW4wggGiMA0GCSqGSIb3DQEBAQUAA4IBjwAwggGKAoIBgQCbe/MqRlv2
+fVHlQRxYJDyqQJuJxloPynLpzTCSylfKa0n+kJLcZvxXBuKREtIgSR7vJzoGtrIs
+74+vxjTYPCcUSYZ2f3rqjX0GvNqQmN25q0r3CW48hlXacMtNCoGLfwChs+qKcDzb
+idwhc47i6qTdNjorQyrj+KCG6gQ8cXEkr6MIEp9DY0LDfm6qmTMXxFBFmojkqLz2
+uyWsWzRu0tccBCkQ5CoxMSGrr//0A60sd+e3X6+CAExKaZyxm06GR3sOzJXRAwGj
+/QrDYlg1V+w/bmA+Bfp/ZIhIgl2zFhGgxDbyIP+EWPhPXx5hJVLOO93906l9kWfk
+U4xsrgCL/CJFHfK9IAcN7X5miOx3gsTrbEg0AOrTW2HrnutWNF/Y1vfTL0pHlYDR
+pzCUs+PhfDr1rmjl1tF/IBjZS8fyEgp+xAqud+1W5Vvp39wpTZtCZAoAuBNyckTv
+KX3jH1AXbz7g6lB9Bi6knf3JKtvr1tBsNWWms3vdDByZUfGkrWK0HD0CAwEAATAN
+BgkqhkiG9w0BAQsFAAOCAYEAh7zwVTLNO8kgE9Nj3cbXAJMEJ6ZoWp3Up+Q19F7x
+Vg3cJt3va/BWmk01ihjr6kXhpPiSZl/+FyQocU5VghxjhTlobzsgKQhsBR0WkBWX
+b6u296J2wybbukuecR000TBUGooHaNetu9KmTS5nW5e7i6tJeztZEg6MndRx1vVD
+bTrKQDqH9eSGj1UcuhT9okRBl6oXavFb8/buweCUptpGfozA8XqX++2UZPo2hR0v
+t1aI2y2LOAb3Iwt997yAI9GIY11uPGJu4sbe4yhpTFM2xh5t3ytPJHTNGk3/poO/
+O9Qt3Dgrp9gCJhZ38gzP+NQwbBOZq6PJXV+4MsRMWuAjYNpiXywmT5z6C362WzEs
+A9/p66zYzpu614GPvQVhkyZ+kpCDD4rqQe3rAu7bWOwfG6q9HFmhpMDloWipLhp0
+p7piYlrotq/Um+Y8tcXxFDTS6Kmi2a3r9g08VP5+qOw0hxwTEImsqSFiy5pGFm7S
+0Lrr3QHpCw+kj3hD8Tap0/iN
+-----END CERTIFICATE-----
```

Comment:
https://github.com/linux-system-roles/logging/blob/master/tests/tasks/create_tests_certs.yml is an example of how a test can generate certs/keys

Reply:
Thanks for the example, fixed in 6ac095a